assets/tests/components/__snapshots__/mapPage.test.tsx.snap

```diff
@@ -1758,12 +1759,13 @@ exports[`<MapPage /> Snapshot renders vehicle data 1`] = `
               </svg>
             </div>
             <div
-              class="leaflet-marker-icon c-vehicle-map__label primary selected leaflet-zoom-hide leaflet-interactive"
+              class="leaflet-marker-icon leaflet-zoom-hide leaflet-interactive"
               role="button"
               style="margin-left: -20px; margin-top: 16px; width: 12px; height: 12px; left: 223px; top: 27px; z-index: 1027;"
               tabindex="0"
             >
               <svg
+                class="c-vehicle-map__label primary selected"
```

Comment:
The way that the `selected` classname moved made me wonder whether the `selected` styles remained the same visually, and it seems they did not. Below are screenshots comparing skate-dev-green and skate-dev, where skate-dev-green no longer has the purple selected state around the vehicle

![Screenshot 2024-10-21 at 7 58 21 AM](https://github.com/user-attachments/assets/2cf282c3-d29b-4ac2-abd3-d2844367f50e)
![Screenshot 2024-10-21 at 7 58 28 AM](https://github.com/user-attachments/assets/190dbda5-431d-4b7f-87f3-969edb93203a)
![Screenshot 2024-10-21 at 7 58 51 AM](https://github.com/user-attachments/assets/55463dad-2831-4dac-a83c-2acb4854b8b8)
![Screenshot 2024-10-21 at 7 58 58 AM](https://github.com/user-attachments/assets/ca67303e-0497-42d4-9351-f21fbd367895)

Reply:
Oooh thanks for catching this! Fixed! https://github.com/mbta/skate/compare/7a0dd0ace1ad2c66235b653f8c682808020fdd66..f78ae953e4fc0e6e346a649e7a73cdcfaf1ff926

Reply:
waiting to confirm from dev-green

Reply:
all good!